ssl-config-generator/index.html

```diff
@@ -334,7 +334,11 @@
                     }
                     if (data.hstsEnabled == "true") {
                         data.hsts = '\n    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)' + '\n' +
-                            '    add_header Strict-Transport-Security max-age=15768000;';
+                            '    add_header Strict-Transport-Security "max-age=15768000"';
```

Comment:
Why has the header string changed as part of this patch? The double quotes have been added and the trailing semicolon removed.